### ISSUE TITLE
Skip accessibility Analyzer

### ISSUE THREAD

**Issue by tweaksalot** [janreges/siteone-crawler-gui#2]
Is there a way to skip the accessibility analyzer - or even most of the analyzers? My scans are getting hung up and when I prematurely stop the scan I see it got stuck in the Analyzer section - mainly the AccessibilityAnalyzer::checkMissingLabels class/method. I don't need accessibility or other analyzers as I'm just looking to crawl and save a site to disk. 

AccessibilityAnalyzer::checkMissingLabels 128 s

**Comment by janreges** [comment]
Hi @tweaksalot,

Yes, you have the possibility to use a regular expression to specify which analyzers you want to run or, on the contrary, which you don't want to run.

It is described more specifically in the documentation of [--analyzer-filter-regex](https://crawler.siteone.io/configuration/command-line-options/#--analyzer-filter-regex), with examples.

You can use e.g. `--analyzer-filter-regex=/^(?:(?!best|access).)*$/i` to prevent the BestPracticeAnalyzer and AccessibilityAnalyzer from running.

I have attached a screenshot where you can insert this regular expression in the GUI.

Btw ... I know, checkboxes would be better - maybe in the future ;)

<img width="287" alt="image" src="https://github.com/user-attachments/assets/c1d9f840-0274-4bbb-a7b3-41dac614235e">
